contracts/mainnet/connectors/pooltogether_v4/main.sol

```diff
@@ -0,0 +1,193 @@
+pragma solidity ^0.7.0;
+
+/**
+ * @title PoolTogether V4
+ * @dev Deposit & Withdraw from PoolTogether V4
+ */
+
+ import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+ import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+ import { PrizePoolInterface, TicketInterface, PrizeDistributorInterface } from "./interface.sol";
+
+import { TokenInterface } from "../../common/interfaces.sol";
+import { Stores } from "../../common/stores.sol";
+import { Events } from "./events.sol";
+import { DSMath } from "../../common/math.sol";
+import { Basic } from "../../common/basic.sol";
+
+abstract contract PoolTogetherV4Resolver is Events, DSMath, Basic {
+    using SafeERC20 for IERC20;
+
+    /**
+     * @dev Deposit into Prize Pool without a delegate
+     * @notice Deposit assets into the Prize Pool in exchange for tokens
+     * @param prizePool PrizePool address to deposit to
+     * @param amount The amount of the underlying asset the user wishes to deposit. The Prize Pool contract should have been pre-approved by the caller to transfer the underlying ERC20 tokens.
+     * @param getId Get token amount at this ID from `InstaMemory` Contract.
+     * @param setId Set token amount at this ID in `InstaMemory` Contract.
+    */
+
+    function depositTo(
```

Comment:
@eccheung4 I think we can name this as `deposit` only? Instead of `depositTo`?

Reply:
yea, should be ok. just thought it might be better to match up the function name with pool together function name.

Reply:
updated to `deposit`